app/static/src/app/store/modelFit/getters.ts

```diff
@@ -1,18 +1,57 @@
 import { Getter, GetterTree } from "vuex";
-import { ModelFitState } from "./state";
+import { ModelFitState, ModelFitRequirements } from "./state";
 import { FitState } from "../fit/state";
+import userMessages from "../../userMessages";
+import { joinStringsSentence } from "../../utils";
 
 export enum ModelFitGetter {
-    canRunFit = "canRunFit"
+    fitRequirements = "fitRequirements"
 }
 
 export interface ModelFitGetters {
-    [ModelFitGetter.canRunFit]: Getter<ModelFitState, FitState>
+    [ModelFitGetter.fitRequirements]: Getter<ModelFitState, FitState>
 }
 
+// I've put this here for want of a better place. Within userMessages
+// feels possible but that would break the default import. Putting it
+// directly in the component makes testing feel much harder. There are
+// tests within getters.test.ts that need moving if this moves.
+export const fitRequirementsExplanation = (reqs: ModelFitRequirements): string => {
```

Comment:
OK, in the end getting getters-from-getters was a bit ugly; partly because vue/vuex makes you fight for it, but also at the point of use it felt like we request a thing then something generated from that thing but via a different getter - even though there are no race conditions it felt a bit weird.

I've moved the support code into the component where we can test it easily and localoise the weird.

The same issue holds for canRunFit, though on a smaller scale